Component/Card/card.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="\css\nav.css">
+    <link rel="stylesheet" href="\css\doc.css">
+    <link rel="stylesheet" href="\Component\Card\card.css">
+    <link rel="stylesheet" href="\css\button.css">
```

Comment:
make a styles.css file and import all the components' css file into them using @import url("").